src/modules/Elsa.Dsl/Interpreters/WorkflowDefinitionBuilderInterpreter/VisitObject.cs

```diff
@@ -38,12 +38,12 @@ public override IWorkflowBuilder VisitObject(ElsaParser.ObjectContext context)
         return DefaultResult;
     }
 
-    private object GetObject(ElsaParser.ObjectContext context)
+    private async Task<object> GetObject(ElsaParser.ObjectContext context)
```

Comment:
Please mind the naming convention to apply the `Async` suffix when changing methods into async.